pyrs/core/peak_collection.py

```diff
@@ -10,21 +10,25 @@ class PeakCollection(object):
     """
     Class for a collection of peaks
     """
-    def __init__(self, peak_tag):
+    def __init__(self, peak_tag, peak_profile, background_type):
```

Comment:
By this change, if there is an another peak fitting effort on the same peak (peak tag) but with different peak/background, a new PeakCollection object is then requested.  I just want to make sure that this is the behavior that you designed.

Reply:
Yes it is. The current `PeakCollection` also only allows for a single out-of-plane angle, which isn't stored.